### PR TITLE
edited icons and add legend

### LINKED ISSUE #359: Public vs Private institution
Problem:
In FORD1 there was an important visual distinction between Public and Private institutions.  

1) there was a legend on the map
2) markers were different
3) it is very important to users
4) it was a primary selection criterion

Have we lost the ability to do this because we have brought data in from the national dbase?

Possible Solution:

If we no longer have this data, we need to add a field in the Provider table so that each record can be flagged public or private.  default value at this stage = 0

We will then populate this field ourselves using value = 1 for public institutions - that list is shorter and more easily acquired.

In the future:  When a new Provider is created this field must be mandatory with no an invalid NULL default value, to force users to populate it

### PR DESCRIPTION
**Pull request by anitanh**
fix #359 

screenshot from 2017-04-04 16-51-49

### CODE REVIEW

File: django_project/feti/api_views/campus.py [hunk 1]
```diff
@@ -62,6 +62,7 @@ def get(self, request, format=None):
                 del stored_fields['long_description']
                 del stored_fields['text']
                 del stored_fields['campus_popup']
+                del stored_fields['campus_public_institution']
```
Comment (dimasciput) [inline django_project/feti/api_views/campus.py]: We need this attribute, don't remove this.

File: django_project/feti/static/js/scripts/views/map.js [hunk 2]
```diff
@@ -176,6 +176,22 @@ define([
...
+                        extraClasses: 'fa fa-graduation-cap',
+                        iconColor: 'white'
+                });
+
+            var legend = L.control({position: 'bottomleft'});
+            legend.onAdd = function (map) {
+                var div = L.DomUtil.create('div', 'legend');
+                div.innerHTML += '<img src="/static/feti/images/marker_public.png">' +'Public Institution </br>';
+                div.innerHTML += '<img src="/static/feti/images/marker_private.png">' + 'Private Institution';
+            return div
```
Comment (dimasciput) [inline django_project/feti/static/js/scripts/views/map.js]: Please fix the indentation

**Review by dimasciput — CHANGES_REQUESTED** [review]
Still a couple of things to fix, also is this only on 'Where to study' tab? What happens in 'What to study'?

**Comment by anitanh** [comment]
this also applies to 'what to study' tab
screenshot from 2017-04-04 19-01-10

although in here we cannot say whether an institution is public or private, as here an institution that was marked as blue in 'where to study' tab was marked as red in 'what to study' tab

**Comment by dimasciput** [comment]
@ann26 so that's still a problem, you need to update the backend for the 'what to study' tab.

**Comment by anitanh** [comment]
updated:
in 'what to study' we can now differentiate public and private

screenshot from 2017-04-05 15-49-10

**Review by dimasciput — APPROVED** [review]

**Comment by dimasciput** [comment]
LGTM

**Pull request merged**